components/src/components/Common/Constants.ts

```diff
@@ -1,3 +1,4 @@
 export abstract class Constants {
   static readonly DEFAULT_HELP_LINK: string = 'https://github.com/bcgov/common-hosted-form-service/wiki';
+  static readonly ADV: string = '';
```

Comment:
I noticed that Constants.ADV was used in all the custom components but is set to an empty string here. Is there any reason for this?